patches/double_field/2091587-text_long-60_2.patch

```diff
@@ -0,0 +1,126 @@
+diff --git a/src/Plugin/Field/FieldFormatter/Base.php b/src/Plugin/Field/FieldFormatter/Base.php
+index d457e31..eee9fb8 100644
+--- a/src/Plugin/Field/FieldFormatter/Base.php
++++ b/src/Plugin/Field/FieldFormatter/Base.php
+@@ -255,6 +255,14 @@ abstract class Base extends FormatterBase {
+           $item->{$subfield} = $field_settings[$subfield][$item->{$subfield} ? 'on_label' : 'off_label'];
+         }
+
++        if ($field_settings['storage'][$subfield]['type'] == 'text_long') {
++          $item->{$subfield} = array(
++            '#type' => 'processed_text',
++            '#text' => $item->{$subfield},
++            '#format' => $item->{$subfield . '_format'},
++          );
++        }
++
+         // Empty string should already be converted into NULL.
+         // @see Drupal\double_field\Plugin\Field\FieldWidget\DoubleField::massageFormValues()
+         if ($item->{$subfield} === NULL) {
+diff --git a/src/Plugin/Field/FieldType/DoubleField.php b/src/Plugin/Field/FieldType/DoubleField.php
+index 43de602..5667c0f 100644
+--- a/src/Plugin/Field/FieldType/DoubleField.php
++++ b/src/Plugin/Field/FieldType/DoubleField.php
+@@ -376,6 +376,15 @@ class DoubleField extends FieldItemBase {
+           $columns[$subfield]['size'] = 'big';
+           break;
+
++        case 'text_long':
```

Comment:
I don't see a switch statement for these cases, and it is not clear which variable is being evaluated against the value 'text_long'. Is this valid?